services-directions-models/src/test/resources/directions_v5_with_toll_costs_and_lanes.json

```diff
@@ -143,10 +143,37 @@
                 }
               ],
               "entry": [
-                true
+                true,
+                true,
+                false
               ],
               "bearings": [
-                198
+                90,
+                180,
+                270
+              ],
+              "geometries": [
+                "k}fiyAcxhgOjCRrD?rDS~CSrDSbQ{@rIg@nFSfES~HSjMSrNRjMz@jHz@jMjCnK~CzJrD~MvG",
+                null,
+                null
+              ],
+              "form_of_way": [
+                "freeway", "ramp", null
+              ],
+              "access": [
+                0,
+                1,
+                null
```

Comment:
`null` is only the possible value for `geometries`. Other fields if present are guaranteed to have a non-null value for all the array elements.